feat: removed descriptions and added info.title as group names for x-tagGroup  in Join command (#1394)
Redocly/redocly-cli: .changeset/grumpy-toys-reply.md

---
"@redocly/cli": patch
---

> Removed descriptions adding for x-tagGroups for the `join` command.

Reviewer: Please include something about why this change was made, and the differences that the user may observe.

---

> "@redocly/cli": patch

Reviewer 2: Changed to minor as requested, thanks!

---

- Removed descriptions adding for x-tagGroups for the `join` command. Descriptions in x-tagGroups are not supported and cause errors on linting.

> - `info.title` is now used as a name in x-tagGroups instead of a file name for the `join` command to make possible the joining of files with the same names.

Reviewer: Will existing users see changed behaviour? Is it configurable?

Reviewer 3: What if the `info.title`'s are the same?

Reviewer 4: Use info versions as well? Is it a valid case to join different versions of the same API descriptions? If even that fails, I'd expect a warning/error. Alternatively, we can add indexes (like Title_1, Title_2 and so on), but this behaviour might be confusing.

Reviewer: If the titles are the same, they'll only get one `x-tagGroups` entry. Let's link to the [info-override decorator](https://redocly.com/docs/cli/decorators/info-override/) in the join command docs so that the user can adjust one or all of the titles to suit their needs before joining.

Reviewer 2: This docs change will be resolved in a separate PR here https://github.com/Redocly/redocly-cli/pull/1422

> - `info.title` is now used as a name in x-tagGroups instead of a file name for the `join` command to make possible the joining of files with the same names.

Reviewer 5: ```suggestion
- Updated `info.title` to be used as a name in x-tagGroups instead of a file name for the `join` command, so you can now join files with the same names.
```

Reviewer 2: Fixed, thanks!
